_posts/en/newsletters/2020-02-26-newsletter.md

```diff
@@ -0,0 +1,138 @@
+title: 'Bitcoin Optech Newsletter #86'
+permalink: /en/newsletters/2020/02/26/
+name: 2020-02-26-newsletter
+slug: 2020-02-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes two proposed routing improvements for
+LN, summarizes three interesting talks from the Stanford Blockchain
+Conference, links to popular questions and answers from the Bitcoin
+StackExchange, and lists several notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.  Note: the list of releases and release candidates has
+been moved to its own [section][release rc section].*
+
+## News
+
+- **Reverse up-front payments:** as described in [Newsletter #72][news72
+  upfront], LN developers have been looking for a way to charge a small
+  fee for routing an LN payment (HTLC) even if the payment is rejected.
+  This can discourage payments that are designed to consume bandwidth
+  and liquidity before ultimately failing at no cost to the sender.
+  This week, Joost Jager [proposed][jager up-front] a new scheme where
+  up-front fees are paid from the receiver of an HTLC back towards the
+  sender of the payment.  For example, if a payment is being sent from
+  Alice to Bob to Carol, then Alice receives a small fee from Bob and
+  Bob receives a small fee from Carol.  The fee would be proportional to
+  the amount of time the HTLC remained pending, which would incentivize
+  quickly routing or rejecting an HTLCs and would ensure users of [hold
+  invoices][topic hold invoices] (for example,  Carol) paid the routing
+  nodes (such as Bob) for tying up their routing capital.
+
+    Several responders seemed to like the idea and began discussing how
+    it might be implemented as well as what technical challenges it
+    would need to overcome.
+
+- **LN direct messages:** Rusty Russell [proposed][russell dm] allowing
+  LN nodes to route encrypted messages between peers without using the
+  LN payments mechanism.  This could replace current uses of
+  messages-over-payments, such as [Whatsat][], with a simpler protocol
+  that might be easier to build upon (such as for the *offers* idea
+  described in [Newsletter #72][news72 offers]).  Russell's proposal
+  originally specified using the same there-and-back onion routing used
+  for LN payments (HTLCs), but developer ZmnSCPxj proposed having the
+  message sender specify the full path from their node to the message
+  recipient and back to the sender.  For example, if Alice wants to
+  communicate with Carol, she might choose the following path:
+
+        Alice → Bob → Carol → Dan → Alice
+
+    This type of circle routing would make surveillance more difficult
+    and would eliminate the overhead for routing nodes of having to
+    store the return path, making the protocol stateless for routing
+    nodes.  Discussion remains ongoing as of this writing, with a focus
+    on preventing the mechanism from being abused for spam and for
```

Comment:
- s/and for/and on/

- perhaps inverse the order of "enhancing privacy" and "preventing ... spam"